src/app/components/modules/simplified-lunch/simplified-lunch.component.ts

```diff
@@ -0,0 +1,124 @@
+import { Component, Input, OnInit, OnDestroy, ElementRef, ViewChild } from '@angular/core';
+import moment from 'moment';
+import * as ElementQueries from 'css-element-queries/src/ElementQueries';
+
+import { AlertService } from '../../../services/alert.service';
+import { LunchService } from '../../../services/lunch.service';
+import { UserService } from '../../../services/user.service';
+
+
+@Component({
+  selector: 'mymicds-simplified-lunch',
+  templateUrl: './simplified-lunch.component.html',
+  styleUrls: ['./simplified-lunch.component.scss']
+})
+export class SimplifiedLunchComponent implements OnInit {
+
+  	loading = true;
+  	lunchDate = moment();
+  	lunch = [];
+    todaysLunch = null;
+  	schools = [
+  		'upperschool',
+  		'middleschool',
+  		'lowerschool'
+  	];
+  	school = this.schools[0];
+    @ViewChild('moduleContainer') containerEl: ElementRef;
+
+  constructor(private alertService: AlertService, private lunchService: LunchService, private userService: UserService) {
+
+  		this.userService.getInfo().subscribe(
+  			data => {
+  				this.school = data.school;
+  			},
+  			error => {
+  				this.alertService.addAlert('warning', 'Warning!', 'We couldn\'t determine your grade. Automatically selected Upper School lunch.', 3);
+  			}
+  		);
+  	}
+
+  ngOnInit() {
+    this.getLunch(moment());
+    ElementQueries.listen();
+		ElementQueries.init();
+  }
+
+  getLunch(getDate) {
+    // Display loading screen
+    this.loading = true;
+
+    this.lunchService.getLunch({
+      year : getDate.year(),
+      month: getDate.month() + 1,
+      day  : getDate.date()
+    }).subscribe(
+      lunch => {
+        console.log(lunch);
+        // Stop loading
+        this.loading = false;
+        // Reset lunch array
+        this.lunch = [];
+        // Get dates for the week
+        let current = moment();
+        let dates = this.getDatesFromWeek(getDate);
+
+        let i = moment().day();
+
+        if (i === 6 || i === 0){
```

Comment:
In general, the MyMICDS codebase does `if` statements like:
```
if ( ... ) {
    ...
} else {
    ...
}
```
There's a space in between the conditional expression and the two curly braces + `else` keyword are on the same line.